### ISSUE TITLE
Version 1.2.0 breaks build_runner

### ISSUE THREAD

**Issue by fennelhans** [dart-lang/logging#143]
```
$ dart run build_runner build --define 'dynamic_config_generator|config_builder=variant=stage' --delete-conflicting-outputs
Building package executable... (3.5s)
Failed to build build_runner:build_runner:
../../../../../.pub-cache/hosted/pub.dev/build_runner_core-7.2.8/lib/src/logging/build_for_input_logger.dart:13:7: Error: The non-abstract class 'BuildForInputLogger' is missing implementations for these members:
 - Logger.onLevelChanged
Try to either
 - provide an implementation,
 - inherit an implementation from a superclass or mixin,
 - mark the class as abstract, or
 - provide a 'noSuchMethod' implementation.

class BuildForInputLogger implements Logger {
      ^^^^^^^^^^^^^^^^^^^
../../../../../.pub-cache/hosted/pub.dev/logging-1.2.0/lib/src/logger.dart:162:22: Context: 'Logger.onLevelChanged' is defined here.
  Stream<Level?> get onLevelChanged {
                     ^^^^^^^^^^^^^^
Failed to build build_runner:build_runner:
../../../../../.pub-cache/hosted/pub.dev/build_runner_core-7.2.8/lib/src/logging/build_for_input_logger.dart:13:7: Error: The non-abstract class 'BuildForInputLogger' is missing implementations for these members:
 - Logger.onLevelChanged
Try to either
 - provide an implementation,
 - inherit an implementation from a superclass or mixin,
 - mark the class as abstract, or
 - provide a 'noSuchMethod' implementation.

class BuildForInputLogger implements Logger {
      ^^^^^^^^^^^^^^^^^^^
../../../../../.pub-cache/hosted/pub.dev/logging-1.2.0/lib/src/logger.dart:162:22: Context: 'Logger.onLevelChanged' is defined here.
  Stream<Level?> get onLevelChanged {
```

**Comment by ditman** [comment]
(I was about to create this very same issue. Thanks!)

**Comment by fennelhans** [comment]
> (I was about to create this very same issue. Thanks!)

I got your back ;)

**Comment by fennelhans** [comment]
For anyone else playing the at-home game, this will get you back up and running until the bug is fixed:

```yaml
dev_dependencies:
  logging: 1.1.1
```

**Comment by musabjaved10** [comment]
Having same issue. Dependency override solves it temporarily

**Comment by devaronius** [comment]
Having the same issue. reverting back to 1.1.1 solved the problem for now

**Comment by ditman** [comment]
I think this can be closed, the fix we've applied elsewhere is to `dart pub upgrade` to ensure that programs are getting the fixed `build_runner_core` version: `7.2.9`

**Comment by acoutts-nydig** [comment]
Thank you for explaining @devoncarew
